docs/querying/query-from-deep-storage.md

```diff
@@ -66,7 +66,9 @@ You can also confirm this through the Druid console. On the **Segments** page, s
 
 Keep the following in mind when working with load rules to control what exists only in deep storage:
 
-- At least one of the segments in a datasource must be loaded onto a Historical process so that Druid can plan the query. The segment on the Historical process can be any segment from the datasource. It does not need to be a specific segment. One way to verify that a datasource has at least one segment on a Historical process is if it's visible in the Druid console.
+- To be queryable, your datasource must meet one of the following conditions:
+  - At least one of the segments in the datasource is loaded onto a Historical process so that Druid can plan the query. The segment on the Historical process can be any segment from the datasource. It does not need to be a specific segment. One way to verify that a datasource has at least one segment on a Historical process is if it's visible in the Druid console.
```

Comment:
```suggestion
  - At least one segment from the datasource is loaded onto a Historical service for Druid to plan the query. This segment can be any segment from the datasource. You can verify that a datasource has at least one segment on a Historical service if it's visible in the Druid console.
```
Changed process to service per https://druid.apache.org/docs/latest/design/historical.